index.html

```diff
@@ -12,11 +12,13 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.13/semantic.min.css"
     />
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed:400,700" 
+    rel="stylesheet" type="text/css">
     <script
       src="https://kit.fontawesome.com/5e7b35dc62.js"
       crossorigin="anonymous"
     ></script>
-    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/style.css"/>
     <title>Discover Serenity</title>
   </head>
   <body>
```

Comment:
Please validate Html file using validator and resolve any error, you can ignore warning if it's not affecting functionality:
One resource:
https://validator.w3.org/
Error: img tag without alt

Reply:
this PR is only focused on the script.js so I may just pull this out and @JonPhoenix work on it or we can fix those with a cleanup issue once we have an MVP so he doesn't have to spend a ton of time working on that and some of it will need to be done in the js (img alt will be set for each trail after getting trail query)